### PR TITLE
feat: Adding a feature for positive and negative bagging fraction params.

### PR DESCRIPTION
**Pull request by ocworld**
Adding a feature for positive and negative bagging fraction params.

Keunhyun Oh <keunhyun.oh@ahnlab.com>

### CODE REVIEW

File: src/main/scala/com/microsoft/ml/spark/lightgbm/LightGBMParams.scala [hunk 1]
```diff
@@ -90,6 +90,18 @@ trait LightGBMParams extends Wrappable with DefaultParamsWritable with HasWeight
...
   def setBaggingFraction(value: Double): this.type = set(baggingFraction, value)
 
+  val posBaggingFraction = new DoubleParam(this, "posBaggingFraction", "Positive Bagging fraction")
+  setDefault(posBaggingFraction->1)
+
+  def getPosBaggingFraction: Double = $(posBaggingFraction)
+  def setPosBaggingFraction(value: Double): this.type = set(posBaggingFraction, value)
+
+  val negBaggingFraction = new DoubleParam(this, "negBaggingFraction", "Negative Bagging fraction")
+  setDefault(posBaggingFraction->1)
```
Comment (mhamilton723) [inline src/main/scala/com/microsoft/ml/spark/lightgbm/LightGBMParams.scala]: This should be neg.

Comment (ocworld) [reply src/main/scala/com/microsoft/ml/spark/lightgbm/LightGBMParams.scala]: Yes, you are right. I've fixed it.

**Comment by imatiach-msft** [comment]
@mhamilton723 seems there is some issue contacting cluster in the E2E tests, is there a known issue currently?

**Comment by imatiach-msft** [comment]
@mhamilton723 has the cluster issue been resolved?  It seems the most recent builds are passing.  Hopefully this should pass.

**Comment by imatiach-msft** [comment]
@mhamilton723 can this PR be merged?  thanks!

**Pull request merged**